owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -242,11 +231,11 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
 
         mCameraUploadsHandler = new CameraUploadsHandler(configuration);
 
-        /**
+        /*
          * Security
          */
 
-        mPrefSecurityCategory = (PreferenceCategory) findPreference("security_category");
+        PreferenceCategory mPrefSecurityCategory = (PreferenceCategory) findPreference("security_category");
```

Comment:
If `mPrefSecurityCategory` is not a class field anymore please rename it to `prefSecurityCategory`. We follow https://source.android.com/setup/contribute/code-style name conventions.

Reply:
This is still pending @hannesa2

Reply:
I simply made a local variable out of it (instead of a field, no renaming)
You gave it the name on 1.3.2018 !

And what's about the other variables ? 
<img width="613" alt="image" src="https://user-images.githubusercontent.com/3314607/51587852-707a0d80-1ee2-11e9-8059-29d70635c9e3.png">

I keep it like it is, we/someone can do this in a next PR

I stop here to discuss codeformat issues, specially which I didn't touched

Reply:
As I told you [above](https://github.com/owncloud/android/pull/2411#discussion_r244693316), we are currently following the code style defined in https://source.android.com/setup/contribute/code-style. Have a look at the image below that appears in the previous link, you have made the class field `mPrefSecurityCategory` a local variable so it can not keep containing the `m` character.  

<img width="579" alt="screenshot 2019-01-23 at 08 36 42" src="https://user-images.githubusercontent.com/15261363/51590403-0d8c7480-1eea-11e9-91fa-cfd1e0bf37bd.png">

> I stop here to discuss codeformat issues, specially which I didn't touched

The point here is that there's nothing to discuss, we have our code style already defined and till we redefine/change our code style, that will be tied to our new Architecture and Kotlin stuff, all the contributors need to respect our current code style and the variable `mPrefSecurityCategory` was modified by you.